css/styles.css

```diff
@@ -0,0 +1,217 @@
+
+/* ---- Base Rules ---- */
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  background: rgb(136, 203, 171);
+  background: linear-gradient(0deg, rgba(136, 203, 171, 1) 0%, rgba(0, 13, 60, 1) 100%);
+  margin: 0;
+  font-family: 'Merriweather', serif;
+  color: #fff;
+}
+
+/* Typeography General */
+h1 {
+  font-family: 'Fira Sans', sans-serif;
+  font-size: 3em;
+  margin: 2em 1rem;
+}
+
+@media only screen and (min-width: 35em) {
+  h1 {
+    font-size: 7em;
+    margin: 2em 4rem 1em;
+  }
+}
+
+h2 {
+  border-bottom: 1px solid #cc1;
+  font-family: 'Oxygen', sans-serif;
+  font-size: 3em;
+  color: #fff;
+}
+
+p {
+  line-height: 1.6em;
+  color: #eee;
+}
+
+/* ---- Layout Rules ---- */
+main {
+  margin: 10vh 1em 10vh;
+}
+
+.main-hero {
+  min-height: 40vh;
+  padding-top: 3em;
+}
+
+section {
+  position: relative;
+  min-height: 80vh;
+}
+
+/* Navigation Styles */
+
+.navbar__menu ul {
+  padding: 2%;
+  padding-left: 0;
+  margin: 0;
+  text-align: right;
```

Comment:
- [ **OPTIONAL** ] The inline comment helps to directly point out the code where the changes are required. So specifying the file name seems redundant. Please kindly remove it. 👍🏽
- [ **OPTIONAL** ] It would have been nice if you could let the author know about the reason the change was requested it helps them to understand the requested changes better. 💯
- [ **OPTIONAL** ] How about using some emojis instead of just the text? 😉